package.json

```diff
@@ -67,6 +67,7 @@
     "react-native-calendar-events": "1.6.4",
     "react-native-callstats": "3.53.4",
     "react-native-fast-image": "5.1.1",
+    "react-native-gifted-chat": "github:FaridSafi/react-native-gifted-chat#938500d7d6185e9d12f11fea6b1cf2e916ca3343",
```

Comment:
If we need a specific commit (do we, tough?) please fork the project into the Jitsi org. We have been bitten by disappearing upstream projects before.